monkey/common/agent_configuration/agent_sub_configuration_schemas.py

```diff
@@ -135,7 +135,7 @@ def _make_exploitation_options_configuration(self, data, **kwargs):
 
 
 class PropagationConfigurationSchema(Schema):
-    maximum_depth = fields.Int()
+    maximum_depth = fields.Int(validate=validate.Range(min=1))
```

Comment:
Depth could be 0. The agent would run everything except the propagation phase.

Reply:
Thought the same but the UI's schema says 1 is the minimum. I'll change both.